paper/paper.md

```diff
@@ -0,0 +1,248 @@
+In the final section of this article, we present a comparison to `ewstools`, showing that
+TransitionsInTimeseries.jl offers a significant speed-up in all the studied cases.
+
+## Reproducibility
+
+Some steps of a transition analysis involve random number generators, which need
+to be handled with care in parallelized codes. This is done in TransitionsInTimeseries.jl,
+which offers the possibility of seeding a random number generator by using the
+keyword argument `rng`, as done in the example shown above. Furthermore,
+TransitionsInTimeseries.jl follows the guidelines of semantic versioning which,
+along with Julia's integrated package manager, ensures that the same code is used for both
+results generation and peer reviewing.
+
+## Reliability
+
+In high-impact context mentioned above, errors are to be avoided with particular care.
+TransitionsInTimeseries.jl is therefore tested via continuous integration on a large test
+suite, thus providing a reliable research framework. Furthermore, a centralized code base
+implies that any new user is a new test, thus increasing the reliability of the code over
+time. Finally, the robustness of the results with respect to a parameter, e.g. the width
+of the sliding window, can be easily studied thanks to the simple syntax, thus contributing
+to the reliability of the results.
+
+## Versatility
+
+TransitionsInTimeseries.jl detaches the degrees of freedom that are available to the user
+from the analysis pipeline. Methods for detection and prediction can thus be applied equally
+well, since both typically rely on the computation of statistical measures over windows of
+the time series. This is of great importance, since a new prediction technique needs to first
+be tested on hindcasting tasks, which requires a reliable detection and timing
+of previous transitions. To illustrate this, a detection task can be performed by merely
+modifying the indicators, change metrics and window type of the code shown above:
+
+```julia
+# Here the data should not be detrended
+time, data = load_data()
+
+indicators = (nothing, nothing)
+change_metrics = (difference_of_mean(), difference_of_max())
+config = SlidingWindowConfig(indicators, change_metrics;
+   width_cha = 50, whichtime = midpoint)
+```
+
+This example showcases that steps of the analysis pipeline can be skipped altogether, as
+done here for the computation of indicators, since the change
+metric of the original time series is already sufficient to detect a transition by comparing
+the difference in mean and maximum values between the two halves of the sliding window.
+Furthermore, the user can define their own `IndicatorChangesConfig` - for instance instead of
+`SlidingWindowConfig` - and are therefore capable of adding new ways of computing the
+relevant metrics. Finally, different ways of testing for significance can be interchangeably
+used, therefore expanding the versatility of the software beyond the surrogate analysis.
+
+## Extensibility
```

Comment:
I am still unhappy about this section. Even though it is extensive, the most extensive subsection, it still does not discuss the biggest novelty of our software: that entirely new pipelines for estimating changes, or estimating their significance, can be added to the software. 

You can save lots of space here by using only two sentences to mention that idicators and change metrics are arbitrary julia functions, which means that they could be defined by the user or imported form any other package.

This extra space can be used to highlight the novelty of new pipelines, like what I contributed in #73 .